packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -21,69 +34,343 @@ import styled from 'styled-components';
+                        paddingBottom={2}
+                        paddingLeft={4}
+                        paddingRight={4}
+                        alignItems="center"
+                        gap={2}
+                        as="button"
+                        hasRadius
+                        onClick={openReleaseModal}
+                      >
+                        <PencilIcon />
+                        <Typography ellipsis>
+                          {formatMessage({
+                            id: 'content-releases.header.actions.edit',
+                            defaultMessage: 'Edit',
+                          })}
+                        </Typography>
+                      </PopoverButton>
+                    </CheckPermissions>
+                    <PopoverButton
+                      paddingTop={2}
+                      paddingBottom={2}
+                      paddingLeft={4}
+                      paddingRight={4}
+                      alignItems="center"
+                      gap={2}
+                      as="button"
+                      hasRadius
+                    >
+                      <TrashIcon />
+                      <Typography ellipsis textColor="danger600">
+                        {formatMessage({
+                          id: 'content-releases.header.actions.delete',
+                          defaultMessage: 'Delete',
+                        })}
+                      </Typography>
+                    </PopoverButton>
+                  </Flex>
+                  <ReleaseInfoWrapper
+                    direction="column"
+                    justifyContent="center"
+                    alignItems="flex-start"
+                    gap={1}
+                    padding={5}
+                  >
+                    <Typography variant="pi" fontWeight="bold">
+                      {formatMessage({
+                        id: 'content-releases.header.actions.created',
+                        defaultMessage: 'Created',
+                      })}
+                    </Typography>
+                    <Typography variant="pi" color="neutral300">
+                      {formatMessage(
+                        {
+                          id: 'content-releases.header.actions.created.description',
+                          defaultMessage:
+                            '{number, plural, =0 {# {unit}s} one {# {unit}} other {# {unit}s}} ago by {createdBy}',
+                        },
+                        { number: -value, unit, createdBy }
+                      )}
+                    </Typography>
+                  </ReleaseInfoWrapper>
+                </Popover>
+              )}
+              <Button size="S" variant="tertiary">
+                {formatMessage({
+                  id: 'content-releases.header.actions.refresh',
+                  defaultMessage: 'Refresh',
+                })}
+              </Button>
+              <Button size="S" disabled={true} variant="default">
```

Comment:
Why is this always disabled and no handler? Is that part of another ticket to publish a release?